packages/core/ssr-handlers.ts

```diff
@@ -21,12 +22,11 @@ export interface SSRHandlersMap {
   updateHTMLAttrs: (selector: string, attribute: string, value: string) => void
 }
 
-const _global = globalThis || this
 const globalKey = '__vueuse_ssr_handlers__'
```

Comment:
I am not a fan of extra deps, use

```ts
const _global = typeof globalThis === 'undefined' ? this : globalThis

Reply:
but the build output `this` will transform to undefined